Models/PFUser+ExtendedUser.m

```diff
@@ -233,6 +234,14 @@ - (void)follow:(PFUser *)user {
             [userRelationship addUserIdToFollowers:self.objectId];
         }];
     }];
+    
+    // create the notification
+    FollowEvent *event = [FollowEvent new];
+    event.user = self;
+    event.eventType = ETFollow;
+    event.followingID = user.objectId;
+    event.place = (Place *)[NSNull new];
```

Comment:
hrmmmm i'm concerned about casting a NSNull to a place. Does creating a new empty place work? or just setting this to nil?